### ISSUE TITLE
[Feature] Suggest relays

### ISSUE THREAD

**Issue by itstomekk** [frnandu/yana#78]
Would be nice if the relays could be suggested by client, so not everything has to be typed

![image](https://github.com/user-attachments/assets/f93bf030-c5fd-47cf-a000-63d6a42e8a71)

**Comment by frnandu** [comment]
It does suggest as you type, but I guess it's covered by the keyboard...should resolve it from an UI point
![Screenshot_20240904_214704_Yana.jpg](https://github.com/user-attachments/assets/21fc4f40-21b6-4aca-b9dd-2c8d03aaacb5)

Or do you mean completely suggested popular relays without you needing to type anything?

**Comment by frnandu** [comment]
@itstomekk ?

**Comment by itstomekk** [comment]
you are right, i didn't notice those after typing, indeed most is covered by keyboard

It's okay! But having them recommended before typing and showing who of your following people uses them might be more helpful for users
